crates/p2p_proto/proto/common.proto

```diff
@@ -9,24 +9,32 @@ message EthereumAddress { bytes elements = 1; }
 message PeerID { bytes id = 1; }
 
 message BlockHeader {
-  FieldElement parent_block_hash = 1;
-  uint64 block_number = 2;
-  FieldElement global_state_root = 3;
-  FieldElement sequencer_address = 4;
-  uint64 block_timestamp = 5;
-
-  uint32 transaction_count = 6;
-  FieldElement transaction_commitment = 7;
-
-  uint32 event_count = 8;
-  FieldElement event_commitment = 9;
-
-  uint32 protocol_version = 10;
+  FieldElement block_hash = 1;
+  FieldElement parent_block_hash = 2;
+  uint64 block_number = 3;
+  FieldElement state_commitment = 4;
+  FieldElement sequencer_address = 5;
+  uint64 block_timestamp = 6;
+  FieldElement gas_price = 7;
+  uint32 transaction_count = 8;
+  FieldElement transaction_commitment = 9;
+  uint32 event_count = 10;
+  FieldElement event_commitment = 11;
+  string starknet_version = 12;
```

Comment:
Dunno what `protocol_version` was for but at least let's retain the information about starknet version when the block was produced.